system/jhtree/jhtree.cpp

```diff
@@ -2614,6 +2663,7 @@ const CJHTreeNode *CNodeCache::getNode(const INodeLoader *keyIndex, unsigned iD,
         cacheEntry = curCache.query(hashcode, &key);
         if (likely(cacheEntry))
         {
+            curCache.numHits->fastInc(1);
```

Comment:
This can be located inside the query() function

Reply:
It could, but query is not overridden or virtual